manager/assets/modext/widgets/core/modx.grid.js

```diff
@@ -169,16 +169,36 @@ MODx.grid.Grid = function(config) {
 
     this.on('click', this.onClickHandler, this);
 };
-Ext.extend(MODx.grid.Grid,Ext.grid.EditorGridPanel,{
+Ext.extend(MODx.grid.Grid, Ext.grid.EditorGridPanel, {
+
     windows: {}
 
+    ,protectedIdentifiers: null
```

Comment:
May be to get it from `$coreContexts` in any way?

Reply:
I'm 99% sure we can not do this. I know what you're getting at and I tried to find a way to not have explicitly state these identifiers in the js, but the event timing poses a problem.